hack/ginkgo-e2e.sh

```diff
@@ -41,6 +41,13 @@ GINKGO_NO_COLOR=${GINKGO_NO_COLOR:-n}
 # If 'y', will rerun failed tests once to give them a second chance.
 GINKGO_TOLERATE_FLAKES=${GINKGO_TOLERATE_FLAKES:-n}
 
+# If set, the command executed will be:
+# - `dlv exec` if set to "delve"
+# - `gdb` if set to "gdb"
+# NOTE: for this to work the e2e.test binary has to be compiled with
+# make WHAT=test/e2e/e2e.test GOGCFLAGS="all=-N -l" GOLDFLAGS=""
```

Comment:
Is there any way to enforce or guarantee this?

Reply:
This is an interesting observation, I think it's possible to use `GOLDFLAGS` inside the program that's run but not from the outside https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-version-information-for-go-applications

I'm not sure if there's a way to get info from the binary generated to enforce this